src/app/shell/workshop-details/side-menu/side-menu.component.html

```diff
@@ -1,7 +1,11 @@
 <div class="container" fxLayout="column"
- [fxLayout.lt-lg]="(isMobileScreen$ | async) ? 'column' : 'row wrap'"
- [fxLayoutAlign]="(isMobileScreen$ | async) ? 'none' : 'center start'">
-  <app-actions *ngIf="role !== Role.provider" [role]="role" [workshop]="workshop"></app-actions>
+ [fxLayout.lt-lg]="isMobileScreen ? 'column' : 'row wrap'"
+ [fxLayoutAlign]="isMobileScreen ? 'none' : 'center start'">
+ <ng-conteiner *ngIf="!isMobileScreen">
+   <app-actions *ngIf="(role !== Role.provider)" [role]="role" [workshop]="workshop"
+     [isMobileScreen$]="isMobileScreen$">
```

Comment:
if you're passing this value to the child component, why I don't see the Input for it? I see you're using selector in actions.ts, then maybe this input should be removed? Otherwise, remove selector.